api/db-cdr/generate-cdr/make-bq-data.sh

```diff
@@ -62,7 +62,7 @@ bq --quiet --project=$BQ_PROJECT query --nouse_legacy_sql \
 VALUES
 (19,0,'Condition','Conditions','Conditions are records of a Person suggesting the presence of a disease or medical condition stated as a diagnosis, a sign or a symptom, which is either observed by a Provider or reported by the patient.',0,0,0),
 (13,3,'Drug','Drug Exposures','Drugs biochemical substance formulated in such a way that when administered to a Person it will exert a certain physiological or biochemical effect. The drug exposure domain concepts capture records about the utilization of a Drug when ingested or otherwise introduced into the body.',0,0,0),
-(21,4,'Measurement','Measurements','Measurement',0,0,0),
+(21,4,'Measurement','Labs and Measurements','Labs and Measurements',0,0,0),
```

Comment:
This will update the domain_info table the next time we generate a new cloud cdr. I'm going to update the table in each env manually so it will be correct until that happens.